test/cmd/createeph.sh

```diff
@@ -59,10 +59,6 @@ os::cmd::expect_failure_and_text "_output/oshinko create_eph sally" "cluster 'sa
 os::cmd::expect_success "oc delete service sally-ui"
 os::cmd::expect_failure_and_text "_output/oshinko create_eph sally" "cluster 'sally' already exists \(incomplete\)"
 
-# exposeui
```

Comment:
This test proably needs to go back, but in a different form. It needs to be replaced with a test that looks at the yaml output and pulls out the route info